internal/backend/s3/s3.go

```diff
@@ -271,12 +271,22 @@ func (be *Backend) Path() string {
 	return be.cfg.Prefix
 }
 
+// Returns whether file should be saved in the provided Storage Class
+func (be *Backend) PassStorageClass(h backend.Handle) bool {
```

Comment:
that method name should start with a lower case letter and probably be called `useStorageClass`